parser/parser.go

```diff
@@ -1580,7 +1579,12 @@ func (p *parser) parseCallOrConversion(fun ast.Expr, isCmd bool) *ast.CallExpr {
 	var list []ast.Expr
 	var ellipsis token.Pos
 	for p.tok != endTok && p.tok != token.EOF && !ellipsis.IsValid() {
-		list = append(list, p.parseRHSOrType()) // builtins may expect a type: make(some type, ...)
+		expr := p.parseRHSOrType()
+		if tuple, ok := expr.(*tupleExpr); ok && isCmd {
```

Comment:
if not isCmd and expr is tupleExpr, what should we do?

Reply:
p.parseRHSOrType(isCmd && len(list) == 0) pass to parseLambdaExpr for check.